src/MSALWrapper/PCAWrapper.cs

```diff
@@ -83,6 +85,20 @@ public interface IPCAWrapper
         /// The <see cref="Task"/>.
         /// </returns>
         Task<TokenResult> GetTokenDeviceCodeAsync(IEnumerable<string> scopes, Func<DeviceCodeResult, Task> callback, CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Tries to get a cached account using the preferred domain if provided.
+        /// </summary>
+        /// <param name="preferredDomain">The preferred domain.</param>
+        /// <returns>The <see cref="Task"/>.</returns>
+        Task<IAccount> TryToGetCachedAccountAsync(string preferredDomain = null);
+
+        /// <summary>
+        /// Tries to get a list of cached accounts using the preferred domain if provided.

```

Comment:
Since this is a "Try*" method, typically that means it might fail. We want to document how this behaves if it fails as well.

Reply:
Added.